HW13/Tests/ElementTests.cs

```diff
@@ -0,0 +1,240 @@
+            ButtonsPage.RightClickOnRightClickMeButton();
+            var actualMessage = ButtonsPage.GetRightClickMessage();
+            Assert.AreEqual(actualMessage, ButtonsPage.rightClickmeButtonMessage);
+        }
+
+        [Test]
+        public void ClickMeButton()
+        {
+            ButtonsPage.SingleClickOnClickMeButton();
+            var actualMessage = ButtonsPage.GetDynamicClickMessage();
+            Assert.AreEqual(actualMessage, ButtonsPage.clickmeButtonMessage);
+        }
+
+    }
+
+    public class RadioButtonTests : BaseTest
+    {
+        readonly RadioButtonPage _RadioButtonPage = new();
+
+        [OneTimeSetUp]
+        public void RadioButtonTestsSetUp() => WebDriverFactory.Driver.Navigate().GoToUrl(TestSettings.DemoQARadioButtonPageUrl);
+
+        [Test]
+        public void CheckThatYesRadioButtonEnabled()
+        {       
+            var actualResult = _RadioButtonPage.IsYesRadioButtonEnabled();
+            Assert.IsTrue(actualResult);         
+        }
+
+        [Test]
+        public void CheckThatNoRadioButtonDisabled()
+        {
+            var actualResult = _RadioButtonPage.IsNoRadioButtonEnabled();
+            Assert.IsFalse(actualResult);
+        }
+
+        [Test]
+        public void CheckThatImressiveRadioButtonEnabled()
+        {
+            var actualResult = _RadioButtonPage.IsImpressiveRadioButtonEnabled();
+            Assert.IsTrue(actualResult);
+        }
+
+        [Test]
+        public void SelectYesRadioButton()
+        {
+            _RadioButtonPage.ClickOnYesRadioButton();
+            var actualMessage = _RadioButtonPage.GetResultMessage();
+            Assert.AreEqual(actualMessage, RadioButtonPage.SelectedYesMessage);
+        }
+
+        [Test]
+        public void SelectImressiveRadioButton()
+        {
+            _RadioButtonPage.ClickOnImpressiveRadioButton();
+            var actualMessage = _RadioButtonPage.GetResultMessage();
+            Assert.AreEqual(actualMessage, RadioButtonPage.SelectedImressiveMessage);
+        }
+
+    }
+
+    public class WebTablesTests : BaseTest
+    {
+        readonly WebTablesPage _webTablesPage = new();
+
+        [OneTimeSetUp]
+        public void WebTablesTestsSetUp() => WebDriverFactory.Driver.Navigate().GoToUrl(TestSettings.DemoQAWebTablesPageUrl);
+
+
+        [Test]
+        public void WebTablesReady() => Assert.IsTrue(_webTablesPage.IsWebTablesPageWorkable());
+        
+        [Test]
+        public void AddNewMember()
+        {
+            _webTablesPage.AddNewMember("Adam", "Smith", "test@test.com", "29", "3000", "test");
+            var actualResult = _webTablesPage.isMemberExistInTable("Adam");
+            Assert.IsTrue(actualResult);
+        }
+    }
+
+    public class LinksTests : BaseTest
+    {
+        readonly LinksPage _LinksPage = new();
+
+        [OneTimeSetUp]
+        public void LinksTestsSetUp() => WebDriverFactory.Driver.Navigate().GoToUrl(TestSettings.DemoQALinksPageUrl);
```

Comment:
you have the same step in each test suite, create OneTimeSetUp in BaseTest and add url as a parameter, then via using base constructor in each class provide url for suite